test/GrantRegistry.t.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.25;
+
+import { Test, console2 } from "forge-std/src/Test.sol";
+import { GrantRegistry, IGrantRegistry } from "../src/GrantRegistry.sol";
+
+contract TestGrantRegistry is Test {
+  GrantRegistry public registry;
+
+  address binance = 0xF977814e90dA44bFA03b6295A0616a897441aceC;
```

Comment:
Why binance? hauhahu Let's usa a more generic name like alice or bob

Reply:
Because it's the official binance wallet

Reply:
We call it Bianco them, he is italian